app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/FormRemoteSettingsPending.tsx

```diff
@@ -19,8 +19,8 @@ const FormApproveConfirm = ({
     <Alert variant="warning">
       <Form className="text-body">
         <p>
-          <strong>Action required —</strong> You need to deploy this experiment
-          to production on Remote Settings.
+          <strong>Action required —</strong> You need to approve this change in
+          Remote Settings.
         </p>
```

Comment:
Must have been missed previously since this isn't part of this task/PR, but I just noticed we're missing a tooltip on the right-hand side here, according to Figma. I don't see what the tooltip should read as though. We can add that as part of the design tweaks issue that'll come out of design review for this.